.github/workflows/publish-runtime.yml

```diff
@@ -22,6 +22,7 @@ on:
           - templates-only
           - simple-template-only
           - frontier-template-only
+          - dancelight-only
```

Comment:
here it's fine because we can select what we want, so it's ok

Reply:
Cool let me know if anything else requires updating for this part of the flow!